src/views/logView.ts

```diff
@@ -32,8 +33,32 @@ export default class LogView extends DocumentView {
 
 export class CommitLongFormItemView extends CommitItemView {
 
-  constructor(public commit: Commit, qualifier?: string) {
+  constructor(public commit: MagitLogCommit) {
     super(commit);
-    this.textContent = `${GitTextUtils.shortHash(commit.hash)} * ${GitTextUtils.shortCommitMessage(commit.message)}`.substr(0, 65).padEnd(70) + `${commit.authorEmail}`;
+    const timeDistance = formatDistanceToNowStrict(commit.time);
```

Comment:
How heavy is this dependency? Going from 0 dependencies hurts, but I assume it will make sense at some point.
But it will need justification :)

Reply:
I thought about that as well. This is importing a specific function, webpack will do tree shaking to only include that function. From the bundle size perspective, that shouldn't be any problem. If you want to look at the bulk of the function. https://github.com/date-fns/date-fns/blob/master/src/formatDistanceStrict/index.js

We can also implement our own as I don't think that's all that difficult, but I use the library because it's well tested, and has exactly thing that we need.

Reply:
sound good 👍